### DISCUSSION TITLE
Preferred editor for working with Markua!

### DISCUSSION THREAD

**Discussion by mglaman** [geerlingguy/ansible-for-devops-manuscript#1] [General]
This is amazing, thank you for releasing this. I have been _trying_ to get into an editorial flow with LeanPub but having issues with Markua and their limited preview builds.

What did you use for your editor? Just VSCode or something else, and deal with the Markua specific markup.

**Comment by geerlingguy** [comment]
Good question! I have written every word for the past decade (maybe longer?) in Sublime Text.

Sometimes on my iPad I'll use Byword, but 99% of the time it's on my laptop or desktop in Sublime.

**Reply by mglaman** [reply]
Thanks! One more question. To edit mobile, was your Git repo inside Dropbox or something? Or does Byword have a GitHub integration

**Reply by geerlingguy** [reply]
Yes, I have all my book files inside Dropbox, so this git repo is one of the folders in there. That way I can edit in pretty much any app that has Dropbox access. I don't typically recommend putting a git repo in Dropbox, but it has worked okay for this simple repo :D

**Reply by mglaman** [reply]
Thanks, as always :)

**Reply by buckeye43210** [reply]
I've started using VS Code with Matt Bierner's Markdown Preview Mermaid Support plugin. I use it at work for SOPs and other technical writing. The previewer supports imbedded (text-based) Mermaid diagrams and it has drag and drop screen shot support :-)
